examples/rainfall/rainfall/model.py

```diff
@@ -0,0 +1,103 @@
+import uuid
+
+import numpy as np
+from mesa import Model
```

Comment:
The minimum mesa version requirement is 1.0.0, and so you can do just `import mesa`. And do `mesa.Model`, etc, for the simplified namespace.

Reply:
Changed this as well.